examples/main.rs

```diff
@@ -9,3 +9,11 @@ fn main() {
     let mut conn = Connection::open_in_memory().unwrap();
     embedded::migrations::runner().run(&mut conn).unwrap();
 }
+
+fn iter_main() {
```

Comment:
since there's no `Cargo.toml` I assumed there was no expectation of actually compiling or running that code. We could make it runnable and put a feature gate around the two methods, but for the purposes of illustration it feels a little extra